### PR TITLE
Added api_helper and account_value example

### PR DESCRIPTION
**Pull request by demianjie**

### CODE REVIEW

File: examples/account/main.go [hunk 1]
```diff
@@ -1,41 +1,41 @@
...
-	if resp.Code != "0" {
-		log.Fatalf("[%s] API Error %s", resp.Code, resp.Message)
-	}
-
-	var data exchange.Balance
-	json.Unmarshal(*resp.Data, &data)
-	yamlFormat, err := yaml.Marshal(data)
-	if err != nil {
-		log.Fatalf("Enable to parse into YAML format: %s | %v", err.Error(), data)
+	if all == true {
```
Comment (metarsit) [inline examples/account/main.go:27]:
```
if all {..}
```

File: examples/account_value/main.go [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+func main() {
+	var key, secret string
+
+	flag.StringVar(&key, "api", "", "API Key Generated by Crypto Exchange")
+	flag.StringVar(&secret, "secret", "", "Secret Key Generated by Crypto Exchange")
+	flag.Parse()
+
+	data, err := exchange.GetAccountAPI(key, secret)
+	if err != nil {
+		os.Exit(1)
```
Comment (metarsit) [inline examples/account_value/main.go:24]: `log.Fatalf` here

File: examples/account_value/main.go [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+		if normal > 0 {
+			value := market[detail.Name+"usdt"] * normal
+			// fmt.Printf("%s\ntotal unit: %.6f\nprice USD : %.2f\n\n", strings.ToUpper(detail.Name), normal, value)
+			fmt.Fprintf(w, "%s,%.6f,%.2f,", strings.ToUpper(detail.Name), normal, value)
+			totalValue += value
+		}
+	}
+	fmt.Fprintf(w, "%.2f\n", totalValue)
+	w.Flush()
+}
```
Comment (metarsit) [inline examples/account_value/main.go:52]: So account value only takes in the consideration of whatever is available, if I were to have an open trade, that means it won't be available in this `account_value` information. Do you think that's required? Or should we make it say.. `available_account_value`? or something?.. I am bad at names... :P

File: examples/account/main.go [hunk 4]
```diff
@@ -1,41 +1,41 @@
...
 	"gopkg.in/yaml.v2"
 )
 
 func main() {
 	var key, secret string
+	var all bool
 
 	flag.StringVar(&key, "api", "", "API Key Generated by Crypto Exchange")
 	flag.StringVar(&secret, "secret", "", "Secret Key Generated by Crypto Exchange")
+	flag.BoolVar(&all, "all", true, "Show all if not only show those has non zero normal values")
```
Comment (metarsit) [inline examples/account/main.go:19]: I think `all` should be `false`? We probably only should show the one we are holding anywhere.

**Comment by demianjie** [comment]
Closing this in favor of tools

**Pull request closed without merging**